### PR TITLE
Delay after resume bugfix

### PR DESCRIPTION
**Pull request by taihun81**
Bugfix for issue #456, and added staggered delay when search resumes.

### CODE REVIEW

**Comment by invisiblek** [comment]
Looks like a couple very minor complaints from travis-ci. Please fix so we can ship!

File: pogom/search.py [hunk 1]
```diff
@@ -258,6 +265,8 @@ def search_worker_thread(args, account, search_items_queue, parse_lock, encrypti
                 sleep_delay_remaining = loop_start_time + (args.scan_delay * 1000) - int(round(time.time() * 1000))
                 if sleep_delay_remaining > 0:
                     time.sleep(sleep_delay_remaining / 1000)
+                else:
+                    time.sleep(args.scan_delay)
```
Comment (invisiblek) [inline pogom/search.py]: So if we missed the window where we should have scanned again, it would wait a full cycle before scanning again?

Comment (taihun81) [inline pogom/search.py]: Ooooh...  My bad, I changed the other line earlier but I forgot to change this one... it should have been  sleep_delay_remaining >= 0:

File: pogom/search.py [hunk 2]
```diff
@@ -292,6 +301,9 @@ def check_login(args, account, api, position):
 
     log.debug('Login for account %s successful', account['username'])
 
+    # To avoid first search step resulting 0/0/0
+    time.sleep(args.scan_delay)
```
Comment (invisiblek) [inline pogom/search.py]: scan_delay timeouts should be handled elsewhere. if we need a pause after logging in, it should be a standard value

Comment (taihun81) [inline pogom/search.py]: This should have been separated PR..  Please ignore and I'm working this out on the other PR request.

**Pull request closed without merging**